backtracking/combination_sum.py

```diff
@@ -37,8 +44,6 @@ def combination_sum(candidates: list, target: int) -> list:
     [[2, 2, 2, 2], [2, 3, 3], [3, 5]]
     >>> combination_sum([2, 3, 6, 7], 7)
     [[2, 2, 3], [7]]
-    >>> combination_sum([-8, 2.3, 0], 1)
```

Comment:
Why remove this test?

Reply:
Because it breaks the constraints and gives "recursion depth exceeded" error. Should i still add?

Reply:
It was breaking the constraints. I have added again.